astropy/table/serialize.py

```diff
@@ -120,11 +120,59 @@ def _represent_mixin_as_column(col, name, new_cols, mixin_cols,
     mixin_cols[name] = obj_attrs
 
 
-def _represent_mixins_as_columns(tbl, exclude_classes=()):
-    """
-    Convert any mixin columns to plain Column or MaskedColumn and
-    return a new table.  Exclude any mixin columns in ``exclude_classes``,
-    which must be a tuple of classes.
+def represent_mixins_as_columns(tbl, exclude_classes=()):
+    """Represent input Table ``tbl`` using only `~astropy.table.Column`
+    or  `~astropy.table.MaskedColumn` objects.
+
+    This function represents any mixin columns like `~astropy.time.Time` in
+    ``tbl`` to one or more plain ``~astropy.table.Column`` objects and returns
+    a new Table.  A single mixin column may be split into multiple column
+    components as needed for fully representing the column.  This includes the
+    possibility of recursive splitting, as shown in the example below.  The
+    new column names are formed as ``<column_name>.<component>``, e.g.
+    ``sc.ra`` for a `~astropy.coordinates.SkyCoord` column named ``sc``.
+
+    In addition to splitting columns, this function updates the table ``meta``
+    dictionary to include a dict named ``__serialized_columns__`` which provides
+    additional information needed to construct the original mixin columns from
+    the split columns.
+
+    This function is used by astropy I/O when writing tables to ECSV, FITS,
+    HDF5 formats.
+
+    Note that if the table does not include any mixin columns then the original
+    table is returned with no update to ``meta``.
+
+    Examples
```

Comment:
`Examples` should be the last section in a docstring...  http://docs.astropy.org/en/stable/development/docrules.html